.env

```
@@ -0,0 +1,3 @@
+MESHY_API_KEY=msy_3ClctZx7svFvzl4kvgXqP2Sa1z7aNIu74dwf
+JULES_API_KEY=AQ.Ab8RN6KwhjKaLXoz4gtOcVKeCiiXfSW-z0czzA4VBR70Y3MgWQ
+FLUID_KIT_ROOT=/Users/jbogaty/assets/Fluid_Kit
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FLUID_KIT_ROOT key should go before the JULES_API_KEY key

(UnorderedKey)